elevenlabs_mcp/server.py

```diff
@@ -139,11 +150,47 @@ def format_diarized_transcript(transcription) -> str:
     except Exception:
         # Fallback to regular text if something goes wrong
         return transcription.text
+@mcp.resource("elevenlabs://{filename}")
+def get_elevenlabs_resource(filename: str) -> Resource:
+    """
+    Resource handler for ElevenLabs generated files.
+    """
+    # Construct the file path
+    output_path = make_output_path(None, base_path)
+    file_path = output_path / filename
+
+    if not file_path.exists():
+        raise FileNotFoundError(f"Resource file not found: {filename}")
+
+    # Read the file and determine MIME type
+    with open(file_path, "rb") as f:
```

Comment:
Should add error handling in case of I/O problems.

```suggestion
  try:
      with open(file_path, "rb") as f:
          file_data = f.read()
  except IOError as e:
      raise FileNotFoundError(f"Failed to read resource file {filename}: {e}")
```

Reply:
sure, added